src/Voron/Platform/Win32/Win32NativeFileMethods.cs

```diff
@@ -87,9 +87,8 @@ public struct FileSegmentElement
             out int pNumberOfBytesRead,
             NativeOverlapped* lpOverlapped
         );
-
-        [DllImport("kernel32.dll", SetLastError = true, CharSet = CharSet.Unicode)]
-
+        
+        [DllImport("kernel32.dll", EntryPoint = "CreateFileW", SetLastError = true, CharSet = CharSet.Unicode, BestFitMapping = false)]
```

Comment:
Why do we need this?

Reply:
This in order to match what we do in Pal also in Win MMap Pager (until it will go into PAL as well)